x-pack/plugins/ml/public/jobs/jobs_list_new/components/utils.js

```diff
@@ -36,6 +36,10 @@ export function isStoppable(jobs) {
   return (jobs.find(j => j.datafeedState === DATAFEED_STATE.STARTED) !== undefined);
 }
 
+export function isClosable(jobs) {
+  return (jobs.find(j => (j.datafeedState === DATAFEED_STATE.STOPPED) && (j.jobState !== JOB_STATE.CLOSED)) !== undefined);
```

Comment:
This could use `jobs.some(...)` and check for `true/false`.

Reply:
far nicer!